lib/rules/sort-keys.js

```diff
@@ -37,6 +37,20 @@ function getPropertyName(node) {
     return node.key.name || null;
 }
 
+/**
+ * Check the Line Separated Groups
+ *
+ * - Compare the start line of the current node with the last line of the previous node.
+ *   If the difference between the two values is greater than 0, Not the same group.
+ * @param {number} thisStartLine is start line of the current node.
+ * @param {number} prevEndLine is end line of the previous node.
+ * @returns {number} number of lines between nodes.
+ * @private
+ */
+function checkLineSeparatedGroups(thisStartLine, prevEndLine) {
```

Comment:
Can we change the function name to `getLinesBetween` or `getNumberOfLinesBetween`?
IMO `check` prefixed name seems to represent `return boolean` or `call context.report()` but actually it returns the number of lines between two.

Reply:
> May I change it to !!Math.max(thisStartLine - prevEndLine - 1, 0); ?

sure :).  it makes sense to me